themes/digital.gov/layouts/partials/core/blog-date-warning.html

```diff
@@ -1,11 +1,24 @@
 {{- $fiveYearsFromToday := now.AddDate -5 0 0 -}}
 {{- if lt .Date $fiveYearsFromToday -}}
-  <div class="usa-alert usa-alert--warning">
+  <div
+    class="usa-alert usa-alert--warning dg-alert dg-alert--out-of-date"
+    role="alert"
```

Comment:
If `role="alert"` is not required on the `uswds` component, should it be? Or is this per project implementation?

Reply:
@nick-mon1 it makes sense in this case because it's warning users that the content below is outdated.